src/chat/fetchingReducer.js

```diff
@@ -51,7 +52,7 @@ export default (state: FetchingState = initialState, action: Action): FetchingSt
       return messageFetchStart(state, action);
 
     case MESSAGE_FETCH_COMPLETE:
-      return messageFetchComplete(state, action);
+      return isSearchNarrow(action.narrow) ? state : messageFetchComplete(state, action);
```

Comment:
These should be parallel to each other.

Reply:
I'm not sure what you mean by 'parallel' here.
Do you mean to bring `? state : mes ...` to it's own line? If so, prettier is automatically bringing them back. 
If not, can you please explain?

Reply:
Sorry, I should have been clearer and used more words. It took me a minute myself just now to figure out what I meant. :wink: 

By "these" I meant the two cases for `MESSAGE_FETCH_START` and `MESSAGE_FETCH_COMPLETE`. They're currently quite parallel -- meaning they behave very similarly, with differences that naturally correspond to the difference between "start" and "complete". So when making changes the changes should continue to treat them in parallel.

Reply:
Thanks for the explanation.

Reply:
Done.